src/e2e-test/resources/pluginParameters.properties

```diff
@@ -70,3 +70,46 @@ pubSubErrorThreshold=0
 pubSubStringValue=one
 pubSubNegativeValue=-100
 ## PUBSUBSINK-PLUGIN-PROPERTIES-END
+
+## BIGQUERYMULTITABLESINK-PLUGIN-PROPERTIES-START
+bqmtMandatoryProperties=referenceName, dataset
+bqmtChunkSize=262144
+bqmtInvalidChunkSize=26214
+bqmtInvalidSinkDataset=$#%$
+bqmtInvalidSinkReferenceName=(*^*&*
+bqmtInvalidTemporaryBucket=$#%$
+bqmtGcsDelimiter=,
+bqmtGcsCsvFile=testdata/BQMT_CSV_TEST.csv
+bqmtGcsCsvFileSchema=[{"key":"zip_code","value":"int"},{"key":"residential_ratio","value":"string"},\
+  {"key":"business_ratio","value":"int"},{"key":"other_ratio","value":"int"},{"key":"total_ratio","value":"string"},\
+  {"key":"tablename","value":"string"}]
+bqmtGcsCsvFileOutputSchema={ "type": "record", "name": "text", "fields": [ \
+  { "name": "zip_code", "type": "int" }, { "name": "residential_ratio", "type": "string" }, \
+  { "name": "business_ratio", "type": "int" }, { "name": "other_ratio", "type": "int" }, \
+  { "name": "total_ratio", "type": "string" }, { "name": "tablename", "type": "string" } ] }
+bqmtGcsTsvFile=testdata/BQMT_TSV_TEST.tsv
+bqmtTsvFileSchema=[{"key":"id","value":"int"},{"key":"company","value":"string"},\
+  {"key":"area","value":"string"},{"key":"tablename","value":"string"}]
+bqmtGcsTextFile=testdata/BQMT_TEXT_TEST.txt
+bqmtGcsTextFileSchema=[{"key":"offset","value":"long"},{"key":"body","value":"string"}]
+bqmtGcsBlobFile=testdata/BQMT_BLOB_TEST.zip
+bqmtGcsBlobFileSchema=[{"key":"body","value":"bytes"}]
+bqmtCreateBQTableQueryFile=testdata/BQMT/BigQueryCreateTableQuery_BQMT.txt
+bqmtInsertBQDataQueryFile=testdata/BQMT/BigQueryInsertDataQuery_BQMT.txt
+bqmtSourceSchemaDatatype=[{"key":"transaction_info","value":"boolean"},{"key":"transaction_num","value":"bytes"},\
+  {"key":"transaction_uid","value":"string"},{"key":"transaction_date","value":"date"},{"key":"transaction_dt",\
+  "value":"datetime"},{"key":"transaction_time","value":"time"},{"key":"latitude","value":"double"},\
+  {"key":"unique_key","value":"long"},{"key":"business_ratio","value":"decimal"},\
+  {"key":"updated_on","value":"timestamp"},{"key":"parent","value":"array"},{"key":"inputs","value":"record"},\
+  {"key":"business_bigratio","value":"decimal"},{"key":"committer","value":"record"},{"key":"trailer","value":"array"},\
+  {"key":"difference","value":"array"},{"key":"Userdata","value":"record"},{"key":"tablename","value":"string"}]
+bqmtCsvTargetTables=ZipcodeOne,ZipcodeTwo,ZipcodeThree
+bqmtDelimitedTargetTables=test,type
+bqmtTextTargetTables=null
```

Comment:
why null ?

Reply:
Add the tables, and tag added with a bug ID.